catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/adapter/BrickAdapter.kt

```diff
@@ -228,7 +238,11 @@ class BrickAdapter(private val sprite: Sprite) :
         for (i in flatItems.indices) {
             adapterPosition = items.indexOf(flatItems[i])
             selectionManager.setSelectionTo(selected, adapterPosition)
-            if (i > 0) {
+            if (checkBoxMode == ALL_DELETE) {
```

Comment:
combine two ifs into one with &, in case it gets to long: make a function which does the check and call it in the if()